helm/scripts/port-forward.sh

```diff
@@ -18,7 +18,7 @@ stop_port_forward() {
 # Start port forwarding for all services
 start_all() {
   start_port_forward "servicex-ben-postgresql" "5432:5432"
-  start_port_forward "servicex-ben-code-gen-uproot" "5001:8000"
```

Comment:
I'm a bit confused about this port forward - is it needed for the func_adl uproot codegen (in which case we should keep the old line too)? In any case the uproot-raw codegen doesn't expose anything on port 5001?